src/main/java/it/gov/pagopa/payhub/pdnd/config/PdndConfig.java

```diff
@@ -0,0 +1,14 @@
+package it.gov.pagopa.payhub.pdnd.config;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "app.pdnd.config")
+@Data
+public class PdndConfig {
+    private String audience;
+    private String key;
```

Comment:
if it's the key pair to the public key already defined on this model, let's specify the name
```suggestion
    private String privateKey;
```